### DISCUSSION TITLE
QTH locator to be pre-fill when creating new QSO is brocken?

### DISCUSSION THREAD

**Discussion by R1BLH** [magicbug/Cloudlog#1765] [General]
> $config['qso_auto_qth'] - Setting this to TRUE allows the QTH locator to be pre-filled based on the person's location when creating new QSO. OSM's Nominatim API is being used for that purpose

It was working before last update. Now there is no this variable in the cloudlog.php configuration file. I tried to add it there, but in gave no result.
QTH locator pre-fill does not work.

**Comment by phl0** [comment]
Check your user profile to en-/disable. See https://github.com/magicbug/Cloudlog/pull/1705

**Reply by R1BLH** [reply]
Wow! It's great! :)
